modules/aws/src/smithy4s/query/FormData.scala

```diff
@@ -0,0 +1,55 @@
+package smithy4s.aws.query
+
+final case class FormData(
+    values: Map[List[String], String],
+    primitive: Option[String]
+) {
+  lazy val render: Option[String] =
```

Comment:
this can be kept a `def`, and return an empty string when it holds a primitive

Reply:
Done